components/ExhibitionsComponents/Exhibition/ImageAndCaption/index.js

```diff
@@ -13,7 +13,6 @@ const ImageAndCaption = ({ exhibition, route }) =>
       <div className={css.imageWrapper}>
         <div
           className={css.image}
-          alt={exhibition.title}
```

Comment:
Just a sanity check here that you want to remove the `alt` text.

Reply:
There was a linter warning because `<div>` can't have `alt`. LMK if there's a better way!

Reply:
yeah, actually, given that exhibition.title is already in a `<p>` just above that, that alt text would just annoy people even if it worked.